app/features/IdentitySlice.ts

```diff
@@ -40,15 +40,17 @@ export const identitiesSelector = (state: RootState) =>
     state.identities.identities;
 
 export const confirmedIdentitiesSelector = (state: RootState) =>
-    state.identities.identities.filter(
-        (identity: Identity) => IdentityStatus.Confirmed === identity.status
-    );
+    state.identities.identities.filter(isConfirmedIdentity);
 
 export const chosenIdentitySelector = (state: RootState) =>
     state.identities.identities.find(
         (i) => i.id === state.identities.chosenIdentityId
     );
 
+export const specificIdentitySelector = (identityId?: number) => (
```

Comment:
It seems odd to have a selector for a specific identity with an optional input. I would expect `identityId` to be required for this selector to make sense.

Reply:
Yea, it was a convenience thing, because I used it in the account report, where the account is not defined from the start. (But I have cleaned up the versioning, so it was not necessary anymore).